### ISSUE TITLE
Questions regarding model inference.

### ISSUE THREAD

**Issue by Strivy-ZSY** [OSUPCVLab/SegFormer3D#28]
Hello, thank you very much for your work! 🥰👍

I have currently matched SegFormer3D to the nnFormer repository (to the best of my ability—it can be trained, though the details may not be entirely accurate). I now want to test the corresponding DSC and HD95 through model inference, which requires the labelsTs. I noticed that the visualization you provided includes a labelsTs folder. May I ask whether this folder contains the actual sample data used for inference testing or just a subset of data for visualization purposes? 🥺  

<img width="917" height="510" alt="Image" src="https://github.com/user-attachments/assets/d185d32d-8fd1-4210-a1a4-fef701800f67" />

Also, in medical image segmentation papers, are the experimental results generally based on the test set, or is it acceptable to use the validation set as well? I am new to this research direction and have added/removed some modules to your model structure. Initially, I evaluated on the validation set, but I feel this might not be rigorous enough. Recently, I have been considering evaluating the experimental results on the test set. ❤️  

Additionally, I saw on GitHub that some people mentioned the Synapse dataset does not include labelsTs and that they used labelsTr for testing. I am unsure about the accuracy of this information and would like to seek your advice. 🥺  

I would greatly appreciate it if you could take the time to reply to these questions amidst your busy schedule! Also, thank you so much for answering some of my questions in the previous email! ❤️❤️❤️
